Packs/VMware/Integrations/VMware/VMware.yml

```
@@ -244,9 +338,288 @@ script:
     - contextPath: VMWare.NICState
       description: VM NIC state (i.e., connected, disconnected, delete).
       type: String
-  dockerimage: demisto/vmware:1.0.0.24293
+  - name: vmware-list-vms-by-tag
+    description: Lists all virtual storage objects attached to the tag.
+    deprecated: false
+    arguments:
+    - name: category
+      description: The category to which the tag belongs.
+      required: true
+      isArray: false
+    - name: tag
+      description: The tag to be queried.
+      required: true
+      isArray: false
+    outputs:
+    - contextPath: VMWareTag.TagName
+      description: The tag that was queried.
+      type: String
+    - contextPath: VMwareTag.Category
+      description: The category to which the tag belongs.
+      type: String
+    - contextPath: VMwareTag.VM
+      description: VM name which the tag attached to.
+      type: String
+  - name: vmware-create-vm
+    description: Creates a new virtual machine in the current folder and attaches
+      it to the specified resource pool. This operation creates a virtual machine,
+      instead of cloning a virtual machine from an existing one.
+    deprecated: false
+    arguments:
+    - name: name
+      description: Display name of the virtual machine.
+      required: true
+      isArray: false
+    - name: cpu-num
+      description: Number of virtual processors in a virtual machine.
+      required: true
+      isArray: false
+    - name: cpu-allocation
+      description: Resource limits for CPU.
+      required: true
+      isArray: false
+    - name: memory
+      description: Resource limits for memory.
+      required: true
+      isArray: false
+    - name: virtual-memory
+      description: Size of a virtual machine's memory, in MB.
+      required: true
+      isArray: false
+    - name: guestld
```

```suggestion
    - name: guest_id
```
same here regarding the naming convention - use underscores and not CamelCase.
note that this was a lower L before.